src/middlewares/error-handler.ts

```diff
@@ -1,8 +1,10 @@
 import {Context} from 'koa';
-import {config} from '../../configurations/configuration';
+import {config} from '../utils/configuration';
+import {logger} from '../utils/logger';
 
 export const errorHandlerMiddleware = async (ctx: Context, next: () => Promise<any>) => {
     try {
+        logger.info(`got ${ctx.request.method}: ${ctx.request.path} request from ${ctx.request.ip}`);
```

Comment:
seperate to logger middleware and error handler middleware

Reply:
done